components/chart/trendlines.md

```diff
@@ -0,0 +1,320 @@
+                new SalesData() { Period = "2021 Q1", Date = new DateTime(2021, 1, 1), Count = 669590 },
+                new SalesData() { Period = "2021 Q2", Date = new DateTime(2021, 4, 1), Count = 793564 },
+                new SalesData() { Period = "2021 Q3", Date = new DateTime(2021, 7, 1), Count = 941133 },
+                new SalesData() { Period = "2021 Q4", Date = new DateTime(2021, 10, 1), Count = 1133020 },
+                new SalesData() { Period = "2022 Q1", Date = new DateTime(2022, 1, 1), Count = 426324 },
+            };
+
+            return data;
+        }
+
+        public static List<SalesData> GenerateScatterData()
+        {
+            var data = SalesData.GenerateData();
+
+            for (int i = 0; i < data.Count; i++)
+            {
+                data[i].NumericPeriod = i + 10;
+            }
+
+            return data;
+        }
+    }
+}
+`````
+
+### Moving Average Trendline
+
+The Moving Average Trendline smoothens out data fluctuations by computing an average of all data points over a specified period. By default, this period is set to `2` chart intervals.
+
+>caption The following example showcases a moving average trendline in line series:
+
+`````CSHTML
+<TelerikChart Width="@Width" Height="@Height">
+    <ChartPannable Enabled="true" Lock="@ChartAxisLock.Y"></ChartPannable>
+    <ChartZoomable Enabled="true"></ChartZoomable>
+    <ChartLegend Visible="true" Position="@ChartLegendPosition.Bottom"></ChartLegend>
+    <ChartTooltip Visible="true" Shared="true"></ChartTooltip>
+
+    <ChartSeriesItems>
+        <ChartSeries Type="ChartSeriesType.Line"
+                     Name="@Series1Name"
+                     Data="@Data"
+                     Field="@nameof(SalesData.Count)"
+                     CategoryField="@nameof(SalesData.Period)">
+            <ChartSeriesMarkers Visible="false"></ChartSeriesMarkers>
+        </ChartSeries>
+
+        <ChartSeries Type="ChartSeriesType.MovingAverageTrendline"
+                     Name="Sales Forecast (MOVING AVERAGE)"
+                     For="@Series1Name"
+                     Color="teal">
+            <ChartSeriesMarkers Visible="false"></ChartSeriesMarkers>
+            <ChartSeriesTrendline>
+                <ChartSeriesTrendline Period="4"></ChartSeriesTrendline>
+            </ChartSeriesTrendline>
+        </ChartSeries>
+    </ChartSeriesItems>
+
+    <ChartCategoryAxes>
+        <ChartCategoryAxis>
+            <ChartCategoryAxisLabels Position="@ChartAxisLabelsPosition.Start">
+                <ChartCategoryAxisLabelsRotation Angle="-90"></ChartCategoryAxisLabelsRotation>
+            </ChartCategoryAxisLabels>
+        </ChartCategoryAxis>
+    </ChartCategoryAxes>
+
+    <ChartValueAxes>
+        <ChartValueAxis Min="-4e5" Max="1.2e6"></ChartValueAxis>
+    </ChartValueAxes>
+</TelerikChart>
+
+@code {
+    public const string Height = "400px";
```

Comment:
Use `private`, according to code style guide.

Consider hard-coding the Width and Height in the Chart if you use them just once and don't change them at runtime.